### DISCUSSION TITLE
Initial configuration from CLI or file

### DISCUSSION THREAD

**Discussion by piotrfura** [globaleaks/globaleaks-whistleblowing-software#4160] [Q&A] [answered]
Hello,
Is it possible to skip the post-installation wizard and perform the initial configuration from the CLI or by providing a configuration file?

Best regards
Piotr

**Comment by evilaliv3** [comment, marked as answer]
Thank you for your question @piotrfura 

No at the moment it is not possible but we are working on a new feature enabling to create configuration profiles so that one will be able to create a tenant starting from preconfigured profiles or eventually initialize the system via a profile file like you are imagining.
